include/aca_config.h

```diff
@@ -18,4 +18,9 @@
 #define MAX_PORT_SCAN_RETRY 300
 #define PORT_SCAN_SLEEP_INTERVAL 1000 // 1000ms = 1s
 
+// prefix to indicate it is an Alcor Distributed Router host mac
+// (aka host DVR mac)
+#define HOST_DVR_MAC_PREFIX "fe:16:11:"
```

Comment:
As discussed, we could move this DVR_MAC_PREFIX to a configuration file later. Do we have an existing configuration file as of now?

Reply:
We don't have an existing configuration file now, we can put everything in this file to it when we create a new one.